packages/client/hmi-client/src/components/stratification/tera-stratification-group-form.vue

```diff
@@ -101,22 +104,26 @@ watch(
 	align-self: stretch;
 }
 
-.first-row {
-	display: flex;
+.subdued-text {
+	color: var(--text-color-subdued);
 }
-.age-group {
+
+.input-row {
+	width: 100%;
 	display: flex;
-	flex-direction: column;
-	padding-right: 2rem;
-	padding-bottom: 1rem;
+	flex-direction: row;
+	flex-wrap: wrap;
+	align-items: center;
+	gap: 0.5rem;
```

Comment:
You can use `var(--gap-small)` for this!
https://github.com/DARPA-ASKEM/terarium/blob/main/packages/client/hmi-client/src/assets/css/theme/_variables.scss#L950
```suggestion
	gap: var(--gap-small);
```

Reply:
done!